### PR TITLE
docs: clarify OTA release precedence

### PR DESCRIPTION
**Pull request by vlindhol**
### Summary

Victor asked a good question at https://memfault.slack.com/archives/C025LBJDQRM/p1697053837450759
so let's clarify why the precedence looks like it does.

### CODE REVIEW

**Comment by vlindhol** [comment]
Current dependencies on/for this PR:
* main
  * **PR #582** <a href="https://app.graphite.dev/github/pr/memfault/memfault-docs/582" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/memfault/memfault-docs/582?utm_source=stack-comment).

File: docs/platform/ota.mdx [hunk 1]
```diff
@@ -235,8 +234,19 @@ The following rules apply to all active Releases per Cohort:
 3. Otherwise, use a Full Release with the highest To Version greater than the
    current version of the device.
 
-In short, the precedence can be summarized like so: **Delta Release** > **MPT
-Release** > **Full Release**.
+:::important
+
+The precedence can be summarized as: **Delta Release** > **MPT Release** >
+**Full Release**.
```
Comment (HBehrens) [inline docs/platform/ota.mdx:240]: I am not sure I'd agree with this summary as it's not precise enough. But then again, when adding detail, it's almost again the list from the paragraph above? 🤔 maybe there's a way to simplify the rules from above?

File: docs/platform/ota.mdx [hunk 2]
```diff
@@ -235,8 +234,19 @@ The following rules apply to all active Releases per Cohort:
...
+
+Due to this precedence, Devices can "skip" must-pass-through Releases via a
+Delta Release. This is because a Delta Release should by definition include all
+information to get from Release `A->B`.
```
Comment (HBehrens) [inline docs/platform/ota.mdx:244]:
```suggestion
Due to this precedence, Devices can "bypass" must-pass-through Releases via a
Delta Release. Due to this precedence, Devices can "bypass" must-pass-through Releases via a
Delta Release. Consequently, a Delta Release must include all
information to get from version `A->B`.
```

**Review by HBehrens — APPROVED** [review]
I like giving an example! Nudged wording a little and corrected use of Version vs. Release.

Instead of repeating the same list of 3 kinds of releases, should we rather try to improve the existing list?

File: docs/platform/ota.mdx [hunk 3]
```diff
@@ -235,8 +234,19 @@ The following rules apply to all active Releases per Cohort:
 3. Otherwise, use a Full Release with the highest To Version greater than the
    current version of the device.
 
-In short, the precedence can be summarized like so: **Delta Release** > **MPT
-Release** > **Full Release**.
+:::important
+
+The precedence can be summarized as: **Delta Release** > **MPT Release** >
```
Comment (vlindhol) [inline docs/platform/ota.mdx:239]:
```suggestion
The precedence can be roughly summarized as: **Delta Release** > **MPT Release** >
```

File: docs/platform/ota.mdx (hunk 1 above)
Comment (vlindhol) [reply docs/platform/ota.mdx:240]: This statement was already there, I just took away some of the "softening" words to make it shorter and punchier. Added a bit more "softness" back!

**Comment by vlindhol** [comment]
### Merge activity

* **Oct 12, 12:03 PM**: @vlindhol merged this pull request with [Graphite](https://app.graphite.dev/github/pr/memfault/memfault-docs/582).

**Pull request merged**